unraid_mcp/tools/info.py

```
@@ -0,0 +1,407 @@
+        if action == "ups_device":
+            variables = {"id": device_id}
+
+        try:
+            logger.info(f"Executing unraid_info action={action}")
+            data = await make_graphql_request(query, variables)
+
+            # Action-specific response processing
+            if action == "overview":
+                raw = data.get("info", {})
+                if not raw:
+                    raise ToolError("No system info returned from Unraid API")
```

Ruff (0.15.0)

[warning] Abstract `raise` to an inner function

(TRY301)